src/CodeGeneration.Roslyn/CompilationGenerator.cs

```diff
@@ -184,7 +184,9 @@ protected virtual Assembly LoadAssembly(string path)
             var loadContext = AssemblyLoadContext.GetLoadContext(this.GetType().GetTypeInfo().Assembly);
             var assembly = loadContext.LoadFromAssemblyPath(path);
 
-            this.dependencyContext = this.dependencyContext.Merge(DependencyContext.Load(assembly));
+            var newDependencyContext = DependencyContext.Load(assembly);
+            if(newDependencyContext!=null)
```

Comment:
Please execute Format Document to fix up whitespace.

Reply:
Sorry I forgot. Should now be formated correctly.

Reply:
I fixed the Naming and Build and test it locally. This time it should Work.

Reply:
By the way for me the xUnit tests are not discoverable (in Visual Studio TestExplorer) untill I update the xunit library and the xunit runner to version 2.3.1.

On the CI it works so I haven't checked those changes in.

*Don't want to break it again*

Reply:
Good to know. I'll send a PR that updates that.